src/test/java/liquibase/ext/spanner/sqlgenerator/CreateDatabaseChangeLogLockTableGeneratorSpannerTest.java

```diff
@@ -0,0 +1,41 @@
+package liquibase.ext.spanner.sqlgenerator;
```

Comment:
nit: would you mind adding (just copy-paste) a copyright header to this file and the other test file?

Reply:
Sure, sorry for missing it.